src/lightning/pytorch/tuner/lr_finder.py

```diff
@@ -456,6 +459,8 @@ def __init__(self, optimizer: torch.optim.Optimizer, end_lr: float, num_iter: in
         self.num_iter = num_iter
         super().__init__(optimizer, last_epoch)
 
+    # mypy can't follow the _TORCH_LRSCHEDULER TypeAlias, so ignore "no base method" error
+    @override  # type: ignore[misc]
     def get_lr(self) -> List[float]:
```

Comment:
Good catch @awaelchli. I missed the inheritance through the `_TORCH_LRSCHEDULER` TypeAlias. So does Mypy, which raises the "no base method" error as a `misc` type error.